Apollonius_graph_2/include/CGAL/Apollonius_graph_2/comparator_profiler.h

```diff
@@ -19,7 +19,6 @@
 
 
 #include <CGAL/Apollonius_graph_2/basic.h>
-#include <CGAL/atomic.h>
```

Comment:
I have a mixed feeling about that.

For the moment the code is using `CGAL::cpp11::atomic`, that is defined in `CGAL/config.h`, but it should be using `std::atomic`, from the header `<atomic>`.

Reply:
I am pretty sure CGAL code is not supposed to use something that is in `CGAL_NO_DEPRECATED_CODE`.

Usages of `<CGAL/atomic.h>` and `<CGAL/thread.h>` should be replaced by usages of proper classes and functions from C++11 `<atomic>` and `<thread>`.